server/channels/app/notification_test.go

```diff
@@ -212,6 +217,309 @@ func TestSendNotifications(t *testing.T) {
 	})
 }
 
+func TestSendNotifications_MentionsFollowers(t *testing.T) {
```

Comment:
I don't see much places we use underscores, In fact we could put this function as a `t.Run()` inside the `TestSendNotifications`. This is just a cosmetic concern, 0/5 so take it as a grain of salt.

Reply:
I split it out into a separate function because the setup for these are different. I suppose we could've nested it deeper, but that didn't feel great. Only 1/5 that I like keeping it in a separate function, but if there's a better naming convention for that, I could change it. I see there's also `TestSendNotificationsWithManyUsers` above which is similar